content/_help/specific-agencies/ssa._es.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: help
+title: SSA.gov
+category: specific-agencies
+permalink: /es/help/specific-agencies/ssa/
+order: 3
+redirect_from:
+---
+Login.gov solo puede contestar preguntas sobre el proceso de inicio de sesión y la creación de una cuenta de Login.gov. Su cuenta de Login.gov no afecta ni contiene información de ningún tipo acerca de su tarjeta, número o prestaciones del [Seguro Social](https://www.ssa.gov/).
+
+No envíe a Login.gov datos confidenciales personales ni números de identificación.
+
+Login.gov solo puede contestar preguntas sobre el proceso de inicio de sesión y la creación de una cuenta de Login.gov.
+
+Si ya tiene una cuenta en [SSA.gov](https://www.ssa.gov/), no necesita crear una nueva en Login.gov a menos que así lo requiera la SSA.
```

Comment:
Since SSA has a Spanish version of their site, I think we should use those links on this page whenever possible, e.g.

```suggestion
Si ya tiene una cuenta en [SSA.gov](https://www.ssa.gov/es), no necesita crear una nueva en Login.gov a menos que así lo requiera la SSA.
```